src/solace_ai_connector/common/log.py

```diff
@@ -18,9 +18,13 @@ def setup_log(logFilePath, stdOutLogLevel, fileLogLevel):
     stream_formatter = logging.Formatter("%(message)s")
     stream_handler.setFormatter(stream_formatter)
 
+    # Create an empty file at logFilePath (this will overwrite any existing content)
+    with open(logFilePath, "w") as file:
+        file.write("")        
+
     # file_handler = logging.handlers.TimedRotatingFileHandler(
     #    filename=logFilePath, when='midnight', backupCount=30, mode='w')
-    file_handler = logging.FileHandler(filename=logFilePath, mode="w")
+    file_handler = logging.FileHandler(filename=logFilePath, mode="a")
```

Comment:
Choose the write mode as `w` (`write`) would close the stream for any follow up log message. Changing to `a` (`append`) keeps the stream open for log stream (Added a manual write to file to clear content)